frontend/src/components/user/AuthenticationGate.tsx

```diff
@@ -1,24 +1,24 @@
 import { getSession } from "src/services/auth/session";
 
+import { ReactNode, Suspense } from "react";
+
+import { RequireLogin } from "./RequireLogin";
+import Loading from "src/components/Loading";
 import { getTranslations } from "next-intl/server";
-import { ReactNode } from "react";
-import { Alert, GridContainer } from "@trussworks/react-uswds";
 
 export async function AuthenticationGate({
   children,
 }: {
   children: ReactNode;
 }) {
+  const t = await getTranslations("LogoutSession")
   const session = await getSession();
-  const t = await getTranslations("Errors");
   if (!session?.token) {
     return (
-      <GridContainer className="margin-top-4">
-        <Alert type="error" heading={t("unauthenticated")} headingLevel="h4">
-          {t("signInCTA")}
-        </Alert>
-      </GridContainer>
-    );
+      <Suspense fallback={<Loading message={t("loading")} />}>
```

Comment:
this is very interesting to me - since there is no async behavior going on in <RequireLogin> I would expect that the fallback would never load here, but it does. Any idea why?

In any case, I'm not sure the brief loading state improves the user experience at all here, so I'd be in favor of removing the suspense unless there's a compelling reason for it that I'm missing.